### ISSUE TITLE
Could not load keyring

### ISSUE THREAD

**Issue by ricardopereira** [orta/cocoapods-keys#49]
### Stack

```
   CocoaPods : 0.36.0
   Ruby : ruby 2.0.0p481 (2014-05-08 revision 45883) [universal.x86_64-darwin14]
   RubyGems : 2.0.14
   Host : Mac OS X 10.10.3 (14D87h)
   Xcode : 6.2 (6C131e)
   Git : git version 2.2.1
   Ruby lib dir : /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib
```

### Plugins

```
cocoapods 0.36.0
cocoapods-keys 1.0.2
cocoapods-plugins 0.4.1
cocoapods-trunk 0.6.0
cocoapods-try 0.4.3
```

### Podfile

```
source 'https://github.com/CocoaPods/Specs.git'

plugin 'cocoapods-keys', {
  :project => "eFatura",
  :keys => [
  ]
}

pod 'Realm'

inhibit_all_warnings!
```

### Error

```
pod install
Analyzing dependencies
Pre-downloading: `Keys` from `https://github.com/ashfurrow/empty-podspec.git`
[!] Could not load keyring
```

**Comment by ashfurrow** [comment]
Ah, I think I see what's happening. You haven't specified any keys, so the keyring isn't created for you (during the pre-installation checks, which require at least one key). If you add a key – any key – and try again, does it still work?

**Comment by ricardopereira** [comment]
Yes, it's working. How can I have a Podfile with no Keys? You told me that with cocoapods-keys it is a requirement to have a record with keys in the Podfile.

**Comment by ricardopereira** [comment]
Thank you :)

**Comment by orta** [comment]
Reopening cause we should show a better error here ( or no show one at all )

I tend towards the better error. I think it's an odd state to include the `keys` but have it empty

**Comment by ashfurrow** [comment]
Cool – sounds like a plan.

**Comment by codeman9** [comment]
I have installed cocoapods-keys for one of my projects with keys and it works great, but when I do a pod install on a project which does not have any keys, i.e., the Podfile doesn't have a cocoapods-keys entry, it errors with the same message.

**Comment by ricardopereira** [comment]
@codeman9 That happened to me too. Then I added some dumb key on the `Podfile` and it worked but not a great solution.

**Comment by nanoxd** [comment]
Had the same thing occur to me with a new project with no cocoapod-keys. Using @ricardopereira fix for now.
